crates/ide/src/syntax_highlighting/escape.rs

```diff
@@ -0,0 +1,25 @@
+use crate::syntax_highlighting::highlights::Highlights;
```

Comment:
You need to add a `//!` doc string here for CI to be happy, just a one liner is fine.